pythonforandroid/bootstraps/service_only/build/src/main/java/org/kivy/android/PythonActivity.java

```diff
@@ -404,7 +404,7 @@ public static void stop_service() {
     }
 
 
-    public static native void nativeSetEnv(String j_name, String j_value);
+    public static native void nativeSetenv(String name, String value);
```

Comment:
Is this required given this is already defined in the parent class `SDLActivity`? I'm not 100% brushed up on my java knowledge but this looks like it might be redundant to me

Reply:
oh right nevermind this is in the non-SDL2 bootstrap, I missed that for a second. all fine :+1: